.github/workflows/aws-staging.yml

```diff
@@ -0,0 +1,54 @@
+on:
+  push:
+    branches:
+      - develop
+
+name: Deploy to AWS Staging
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: eu-central-1
+      S3_BUCKET: oasis-casual-staging
+      CF_DISTRIBUTION_ID: E3SWP538EXVDQ5
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Setup Node Version
+      uses: actions/setup-node@v1
```

Comment:
@jeannettemcd there's now `v2` available

Reply:
That is the correct version for the node version that is required by the app. The older versions of node cache differently than the newer versions, so the appropriate version of setup-node has to be used.